ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -448,9 +449,21 @@ class Measure extends Annotation {
     }
 
     public addControlPoints(): void {
+        const options = this.options.controlPointOptions;
         let selectType = this.options.typeOptions.selectType,
             controlPoint;
 
+        if (
+            !defined(this.userOptions.controlPointOptions &&
+                this.userOptions.controlPointOptions.style.cursor)
+        ) {
```

Comment:
I was wondering if we could improve the bracket position here to make it a little bit more readable, maybe like this?:
```js
if (!defined(
    this.userOptions.controlPointOptions &&
    this.userOptions.controlPointOptions.style.cursor
)) {
```